admin-part/src/main/kotlin/Plugin.kt

```diff
@@ -309,8 +309,27 @@ class Plugin(
         Routes.Build.Summary(buildRoute).let {
             send(buildVersion, Routes.Build.Summary.TestsToRun(it), testsToRun.toSummary())
         }
-        send(buildVersion, Routes.Build.MethodsCoveredByTest(buildRoute), methodsCoveredByTest)
-        send(buildVersion, Routes.Build.MethodsCoveredByTestType(buildRoute), methodsCoveredByTestType)
+        //TODO remove after changes on the frontend
+        send(buildVersion, Routes.Build.CoveredMethodsByTest(buildRoute), methodsCoveredByTest)
+        send(buildVersion, Routes.Build.CoveredMethodsByType(buildRoute), methodsCoveredByType)
```

Comment:
Add newline after this one. Without it there is no clue what to remove after changes on the front.